Sources/Drain.swift

```diff
@@ -22,7 +22,7 @@ public final class Drain: DataRepresentable, Stream {
 
 
         while !stream.closed {
-            if let chunk = try? stream.receive() {
+            if let chunk = try? stream.receive(max: Int.max) {
                 buffer.bytes += chunk.bytes
```

Comment:
We could extend `Int`.

Maybe something like `Int.maxStreamChunkSize` ? That's a little verbose.